src/components/Book.css

```diff
@@ -0,0 +1,139 @@
+#root {
+  padding: 0;
+}
+
+.book {
+  display: flex;
+  justify-content: space-between;
+  margin: 6rem;
+}
+
+.title-book {
+  color: gray;
+}
+
+.Book {
+  display: flex;
+  gap: 8rem;
+  margin: 0 6rem;
+  align-items: center;
+  margin-top: 4rem;
+  border: 1px solid lightgray;
+  padding: 2rem;
+}
+
+.porcent {
+  margin: 0;
+  font-weight: normal;
+}
+
+.porcent-name {
+  margin: 0;
+}
+
+.div-form {
+  display: flex;
```

Comment:
- [ ]  It would be better if we implement the CSS best practices. The property display: flex appears more than once. So we can fix it in many ways, but i will highlight on one. .class, .class2 {display: flex }. Kindly do these for all other repeating rules